plugin.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Memphis Export",
+    "name": "Memphis.dev",
```

Comment:
Hello @joethreepwood thank you for the review. I am wondering if **Memphis Broker** would give more context to the app, instead of **Memphis Export**, if you still think that the latter would be a better fit, then I don't mind going with it.

Reply:
Hey @EbubeCode I do think Memphis Export is a bit clearer, as well as fitting with our convention. We tend to call apps like this Import (data in), Export (Data out), Connector (two-way).

Reply:
Okay @joethreepwood I just renamed it, thank you.